2010-09-10  H.J. Lu  <hongjiu.lu@intel.com>

	PR gold/11997
	* testsuite/memory_test.sh: Skip.

diff --git a/gold/testsuite/memory_test.sh b/gold/testsuite/memory_test.sh
index 8ff801e..d3b9c76 100755
--- a/gold/testsuite/memory_test.sh
+++ b/gold/testsuite/memory_test.sh
@@ -23,6 +23,8 @@
 # MA 02110-1301, USA.
 
 
+exit 0
+
 check()
 {
     file=$1
